workitem/link/category-repository.go

```diff
@@ -13,11 +13,11 @@ import (
 
 // WorkItemLinkCategoryRepository encapsulates storage & retrieval of work item link categories
 type WorkItemLinkCategoryRepository interface {
-	Create(ctx context.Context, name *string, description *string) (*app.WorkItemLinkCategory, error)
-	Load(ctx context.Context, ID string) (*app.WorkItemLinkCategory, error)
-	List(ctx context.Context) (*app.WorkItemLinkCategoryArray, error)
+	Create(ctx context.Context, name *string, description *string) (*app.WorkItemLinkCategorySingle, error)
```

Comment:
You wouldn't have needed these changes hadn't the Database layer dependent on the REST layer.. :)